### ISSUE TITLE
Black screen when running s&box in Vulkan

### ISSUE THREAD

**Issue by ElmarTalibzade** [Facepunch/sbox-issues#4879]
### Describe the bug

Entire monitor turns black whenever I attempt interacting with s&box editor. The only way to fix this is to run using `-dx11` flag.

### To Reproduce

1. Run s&box
2. Try interacting with it

### Expected behavior

It shouldn't behave like that

### Media/Files

I've attached log file.
[Log.log](https://github.com/Facepunch/sbox-issues/files/14336227/Log.log)

I've decided to capture the video from my phone camera as it's easier to see the bug in action.

https://github.com/Facepunch/sbox-issues/assets/8478519/6792f89d-2ba2-40cd-b703-592c12d6a10d

### Additional context

OS: Windows 10 Pro 19045.4046
Video card: Nvidia GeForce RTX 3080 FE 10GB
Processor: AMD Ryzen 5 5600X 6-Core Processor 3.70 GHz
RAM: 32.0 GB

All drivers are up to date.

**Comment by handsomematt** [comment]
Sorry to be a pain, but could you try these debug steps:

* Checking Event Viewer for GPU events
* Make sure you have no third party overlays installed
* Try disabling the Steam Overlay
* Downgrading drivers to 537.58, recent releases seem to have a lot of complaints around them on Reddit

**Comment by ElmarTalibzade** [comment]
Hi Matt,

Downgrading to 537.58 didn't fix the issue.
In the Event Viewer, is there a particular log group that you'd be interested in?

Steam Overlay was disabled for s&box. I have no other overlays installed.

I'm attaching generated vulkan infos (both for old and latest driver):
- [vulkaninfo - 551.52.txt](https://github.com/Facepunch/sbox-issues/files/14336879/vulkaninfo.-.551.52.txt)
- [vulkaninfo - 537.58.txt](https://github.com/Facepunch/sbox-issues/files/14336880/vulkaninfo.-.537.58.txt)

**Comment by handsomematt** [comment]
Debugging steps with @ElmarTalibzade - Game works, only editor is broken, doesn't break until you interact with it(?)

* Disabling all layers (EOS, Overwolf)
* Updating Vulkan SDK
* Disabling gsync
* Checking for validation errors
  * ```Validation Error: [ VUID-vkCmdEndQuery-None-07007 ] Object 0: handle = 0x26bca8fb3c0, type = VK_OBJECT_TYPE_COMMAND_BUFFER; Object 1: handle = 0x925c750000001796, type = VK_OBJECT_TYPE_QUERY_POOL; Object 2: handle = 0xd7cd1e00000016ef, type = VK_OBJECT_TYPE_RENDER_PASS; | MessageID = 0x1f270e65 | vkCmdEndQuery():  query (0) was started outside a renderpass.```
  Seems unrelated 

What seems to have worked is either setting the refresh rate and max_fps to 60, hopefully @ElmarTalibzade can help narrow this down some more soon.

**Comment by ElmarTalibzade** [comment]
I've narrowed down the problem to the Scene window.

- When the engine loads up, I get black screen issues *until* I switch to Game window.
- While the screen is black, I can still interact with UI, letting me do that change windows
- After that, everything works as normal: no black screen problems. I can even switch back to Scene window after that.

I was able to confirm this by changing my layout so that Game window gets rendered first. Restarting the engine with that layout eliminated the problem entirely.

This is all given that overlays are enabled (except Steam Overlay), my monitors run at 165HZ, there's no FPS cap etc.
